doc/nrf/libraries/others/event_manager.rst

```diff
@@ -142,6 +142,10 @@ To create a source file for the event type you defined in the header file:
 #. Define the event type with the :c:macro:`EVENT_TYPE_DEFINE` macro.
    Pass the name of the event type as declared in the header along with additional parameters.
    For example, you can provide a function that fills a buffer with a string version of the event data (used for logging).
+   The :c:macro:`EVENT_TYPE_DEFINE` macro adds flags as a last parameter.
+   These flags are constant and can only be set using :c:macro:`EVENT_FLAGS_CREATE` on :c:macro:`EVENT_TYPE_DEFINE` macro.
+   If you do not want to set any flag use :c:macro:`EVENT_FLAGS_CREATE` without any argument as shown in a example below.
```

Comment:
```suggestion
   If you do not want to set any flag, use :c:macro:`EVENT_FLAGS_CREATE` without any argument as shown in the below example.
```
Just minor updates

Reply:
I think it should be: "To **not** set any flag...."